lib/pfconfig/namespaces/resource/ViolationFilterEngine.pm

```diff
@@ -0,0 +1,112 @@
+package pfconfig::namespaces::resource::ViolationFilterEngine;
```

Comment:
Should be renamed to lib/pfconfig/namespaces/FilterEngine/ViolationFilterEngine.pm